src/Conda.jl

```diff
@@ -70,7 +70,16 @@ function _set_conda_env(cmd)
         pop!(env, var)
     end
     env["CONDARC"] = CONDARC
-    setenv(cmd, env)
+    env
+end
+
+"""
+Use a cleaned up environment for the command `cmd`.
+
+Any environment variable starting by CONDA will interact with the run.
+"""
+function _set_conda_env(cmd)
```

Comment:
changed something,
but it still needs the `_set_conda_env` function for
that is used to get the information from the commands.

```
 return collect(keys(JSON.parse(readstring(_set_conda_env(`$conda search $package --json`)))))
```